### ISSUE TITLE
Something about the conversion is wrong

### ISSUE THREAD

**Issue by yaron-tracxpoint** [pieterlange/aws-dia-shapes#3]
For some reasons the icons in Dia render incorrectly
![screenshot_2017-08-21_10-36-24](https://user-images.githubusercontent.com/31091476/29508296-a871e8d6-865c-11e7-8a26-3fde296b22bd.png)
.

**Comment by pieterlange** [comment]
Yeah, i've noticed that before too, but couldn't be bothered to fix it as i had the icons i needed. (not all icons produce artifacts like these)

Feel free to dig into it yourself, i won't be fixing this any time soon.

**Comment by yaron-tracxpoint** [comment]
Sure, fair enough :)
